aicsimageio/readers/__init__.py

```diff
@@ -1,6 +1,41 @@
 #!/usr/bin/env python
 # -*- coding: utf-8 -*-
 
-from .array_like_reader import ArrayLikeReader  # noqa: F401
-from .ome_tiff_reader import OmeTiffReader  # noqa: F401
-from .tiff_reader import TiffReader  # noqa: F401
+from typing import TYPE_CHECKING, Type
+
+# To add a new reader add it both to TYPE_CHECKING and _READERS
+
+if TYPE_CHECKING:
+    from .array_like_reader import ArrayLikeReader  # noqa: F401
```

Comment:
side-note: we could also get rid of these `# noqa: F401` by adding each string literal to the `__all__` section below (e.g. `__all__ = ['ArrayLikeReader', ...]`, but that adds yet another place to add each reader, instead of just adding to `_READERS`

Reply:
Yea. I tend to like just having it in once place rather than both. The noqa doesn't bother me too much.